03-nodejs/index.js

```diff
@@ -29,4 +29,45 @@ const app = express();
 
 // TODO
 
+//1 - create the GET Users endpont
+app.get('/users', function(req, res) {
+  //Set the response header to download csv file
+  usersCall((err, results) => {
+    if (err) {
+      res.send('error');
+    }
+    res.setHeader('Content-Type', 'text/csv');
+    res.setHeader(
+      'Content-Disposition',
+      'attachment; filename="' + 'users-' + Date.now() + '.csv"'
+    );
+    res.send(formatCSVOutput(results));
+  });
+});
+
```

Comment:
This functions works, but could have been implemented in ES6, at least as Arrow functions